static/js/app.js

```diff
@@ -128,8 +136,7 @@ $('#label-list').on('click', '.colorChange', function () {
 
 //change label color
 $('#label-list').on('change', '.colorChangePicker', function () {
-  //TODO: specify original color and label
-  console.log('Highlight color changed to: ' + this.value);
```

Comment:
need to keep this and expand on it a bit in the future

Reply:
fixed in latest commit